scripts/download.pl

```diff
@@ -277,21 +276,18 @@ sub cleanup
 		push @mirrors, "https://mirror.netcologne.de/gnu/$1";
 		push @mirrors, "http://ftp.kddilabs.jp/GNU/gnu/$1";
 		push @mirrors, "http://www.nic.funet.fi/pub/gnu/gnu/$1";
-		push @mirrors, "http://mirror.internode.on.net/pub/gnu/$1";
 		push @mirrors, "http://mirror.navercorp.com/gnu/$1";
```

Comment:
I think we can here use HTTPS instead of HTTP, right?

Reply:
Not all of those sites do support https, so even some of my fixes were just http.

But I could do a second evaluation, and try to switch as many http sites to https, if that is preferred.

Reply:
From my side, we should really switch those websites from HTTP to HTTPS, if they support it because it increases security and also would be a good addition to the already merged commit - https://github.com/openwrt/openwrt/commit/90c6e3aedf167b0ae1baf376e7800a631681e69a